plugin/klogrus/klogrus.go

```diff
@@ -45,6 +61,8 @@ func kgoToLogrusLevel(level kgo.LogLevel) (logrus.Level, bool) {
 		return logrus.InfoLevel, true
 	case kgo.LogLevelDebug:
 		return logrus.DebugLevel, true
+	case kgo.LogLevelNone:
+		return logrus.TraceLevel, false
```

Comment:
No reason to explicitly add this case, since the return just below is the same thing?